packages/calypso-e2e/src/lib/flows/start-import-flow.ts

```diff
@@ -94,6 +94,13 @@ export class StartImportFlow {
 		);
 	}
 
+	/**
+	 * Validates that we've landed on the import (migration) ready page.
+	 */
+	async validateImportReadyPage(): Promise< void > {
+		await this.page.waitForSelector( selectors.startBuildingHeader( 'You are ready to migrate' ) );
```

Comment:
**issue:soft-blocking** - `waitForSelector` is no longer recommended by upstream; `locator` or an accessible locator, such as `getByRole` is better. 

I won't block a merge in this, but it's preferable to use modern methods.

Reply:
Thanks @worldomonation,

In this PR we updated the flow, and we also need to update the e2e test, which fails now. I've followed the same selector approach we already had in the file. Maybe we can get rid of bad practices in the separate PR.

Reply:
I understand that. Generally speaking, I try to leave whatever I touch in a better state than befere, but I get this isn't your area of expertise either.

An accessible selector could be something like `page.getByText( 'You are ready to migrate' ), which is the same as the current code, without needing to define a CSS selector.